metadata.js

```diff
@@ -39,14 +40,15 @@ export const updateMetadata = ({title, description, url, image}) => {
     _setMeta('property', 'og:description', description);
     _setMeta('property', 'twitter:description', document.title);
```

Comment:
In reviewing this PR I found a bug in the above line which should be setting to `description` instead of `document.title`. Mind fixing in this PR too?